examples/storage/cassandra/README.md

```diff
@@ -30,7 +30,7 @@ This example also uses some of the core components of Kubernetes:
 - [_Pods_](../../../docs/user-guide/pods.md)
 - [ _Services_](../../../docs/user-guide/services.md)
 - [_Replication Controllers_](../../../docs/user-guide/replication-controller.md)
-- [_Pet Sets_](http://kubernetes.io/docs/user-guide/petset/)
+- [_StatefulSets_](http://kubernetes.io/docs/user-guide/statefulset/)
```

Comment:
We don't have `http://kubernetes.io/docs/user-guide/statefulset/` yet. To do this we need to rename the file in the docs repo and redirect http://kubernetes.io/docs/user-guide/petset/ to http://kubernetes.io/docs/user-guide/statefulset/ 

We can leave this link as-is for now